content/docs/code-splitting.md

```diff
@@ -145,7 +146,7 @@ function MyComponent() {
 }
 ```
 
-The `fallback` prop accepts any React elements that you want to render while waiting for the component to load. You can place the `Suspense` component anywhere above the lazy component. You can even wrap multiple lazy components with a single `Suspense` component.
+The `fallback` prop accepts any React elements that you want to render while waiting for the component to load. You can place the `Suspense` component anywhere above the lazy component. You can even wrap multiple lazy components with a single `Suspense` component. All of them will be rendered same time and this is useful to show only single loading indicator, instead of showing multiple loading indicator for invidual component.
 
```

Comment:
Added explanation about what will happen in race condition.(One lazy loaded component load faster/slower than other)
[reference](https://web.dev/code-splitting-suspense/)